src/libraries/Microsoft.Extensions.Http/src/Logging/LogHelper.cs

```diff
@@ -25,11 +25,15 @@ private static class EventIds
             public static readonly EventId RequestHeader = new EventId(102, "RequestHeader");
             public static readonly EventId ResponseHeader = new EventId(103, "ResponseHeader");
 
+            public static readonly EventId RequestFailed = new EventId(104, "RequestFailed");
+
             public static readonly EventId PipelineStart = new EventId(100, "RequestPipelineStart");
             public static readonly EventId PipelineEnd = new EventId(101, "RequestPipelineEnd");
 
             public static readonly EventId RequestPipelineRequestHeader = new EventId(102, "RequestPipelineRequestHeader");
             public static readonly EventId RequestPipelineResponseHeader = new EventId(103, "RequestPipelineResponseHeader");
+
+            public static readonly EventId PipelineFailed = new EventId(104, "RequestPipelineRequestFailed");
```

Comment:
To align with PipelineStart and PipelineEnd event IDs:
```suggestion
            public static readonly EventId PipelineFailed = new EventId(104, "RequestPipelineFailed");
```

Reply:
@CarnaViire done